src/main/java/org/kafkahq/modules/KafkaWrapper.java

```diff
@@ -167,4 +170,23 @@ public DescribeClusterResult describeCluster(String clusterId) throws ExecutionE
             names
         );
     }
+
+    public Collection<AclBinding> describeAcls(String clusterId, AclBindingFilter filter) throws ExecutionException, InterruptedException {
+        return Lock.call(() -> {
+                    try {
+                        return kafkaModule.getAdminClient(clusterId)
+                                .describeAcls(filter)
+                                .values()
+                                .get();
+                    } catch (ExecutionException e) {
+                        if (e.getCause() instanceof SecurityDisabledException) {
```

Comment:
```
                        if (e.getCause() instanceof SecurityDisabledException | e.getCause() instanceof ClusterAuthorizationException) {
```
If user don't have ACL, it will crash KafkaHQ